examples/p4est_2d_dgsem/elixir_euler_supersonic_cylinder_amr.jl

```diff
@@ -0,0 +1,139 @@
+# Compressible Euler flow around a cylinder at Mach 3
```

Comment:
I don't think we need the `_amr` suffix in the elixir name? IIRC, we discussed the naming scheme and decided to include it only if it's an important feature, e.g., if we have something like `elixir_euler_isentropic_vortex.jl` and `elixir_euler_isentropic_vortex_amr.jl`.

Reply:
Done, the file name doesn't have `amr` anymore